docs/manual-CN/4.contributions/pull-request-commit-message-guidelines.md

```diff
@@ -0,0 +1,16 @@
+# PR 和 Commit Message 指南
+
+本文档介绍的 PR 和 Commit Message 指南适用于所有 **Nebula Graph** 仓库。 所有提交至 `master` 分支的 commit 均必须遵循以下准则。
+
+## Commit Message
+
+- 首行 subject 是 commit 的简短描述，不超过 50 个字符。
+- 如需添加详细信息，请添加空白行，然后以段落格式进行添加。
+
+## Pull Request
+
+提交 PR 时，请在标题中包含所有更改的详细信息，并确保标题简洁。
+
+PR 标题必需简明概括更改信息。
+
+对于显而易见的简单更改，可不必添加描述。如果 PR 涉及复杂更改，请对更改进行概述。如果 PR 修复了相关 issue，请关联。
```

Comment:
I prefer if you can add an example for every line.
I'm a bit impatient to read the guidelines.